content/docs/user-guide/serving/streamlit.md

```diff
@@ -0,0 +1,58 @@
+# Streamlit
+
+[Streamlit](https://streamlit.io) is an open-source Python library that makes it
+easy to create and share beautiful, custom web apps for machine learning and
+data science.
+
+MLEM can use Streamlit to expose your model to external users with a nice
+auto-generated UI.
+
+## Requirements
+
+```bash
+pip install mlem[streamlit]
+# or
+pip install fastapi uvicorn streamlit streamlit_pydantic
```

Comment:
Let's try to be consistent now that https://github.com/iterative/mlem.ai/pull/268 is in 🙂 

````suggestion
```cli
$ pip install mlem[streamlit]
# or
$ pip install fastapi uvicorn streamlit streamlit_pydantic
````

Reply:
p.s. looks like lots of `$` are still missing in other places ([e.g.](https://mlem.ai/doc/user-guide/serving/rabbitmq))... Didn't catch that in #268.